uuid/test_uuid.py

```diff
@@ -0,0 +1,29 @@
+import uuid
+
+# Add some extra interface to UUID for testing
+class UUID(uuid.UUID):
```

Comment:
As far as I can tell, this "testing" class is now a dead (unexecuted) code, which now contains error. Can you please remove it completely then?